2019-12-07 20:15:30,135 manager INFO 1.1.0
2019-12-07 20:15:30,135 manager INFO Copyright (C) 2003-2010
2019-12-07 20:15:30,135 manager INFO   Noriaki Ando
2019-12-07 20:15:30,135 manager INFO   Intelligent Systems Research Institute, AIST
2019-12-07 20:15:30,135 manager INFO Manager starting.
2019-12-07 20:15:30,135 manager INFO Starting local logging.
2019-12-07 20:15:30,135 manager.namingmanager INFO NameServer connection succeeded: corba/localhost
2019-12-07 20:15:32,157 ManagerServant INFO Master manager not found
2019-12-07 20:15:32,160 manager WARNING Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OpenRTM_aist\Manager.py", line 1900, in initManagerServant
    reffile = file(self._config.getProperty("manager.refstring_path"),'w')
IOError: [Errno 2] No such file or directory: '/var/log/rtcmanager.ref'

2019-12-07 20:15:32,161 LocalServiceAdmin INFO All the local services are enabled.
2019-12-07 20:15:32,173 manager INFO Component type conf file: Change.conf loaded.
2019-12-07 20:15:32,180 ec_base INFO setRate(1000000.000000) done
2019-12-07 20:15:32,180 ec_base INFO setRate(1000.000000) done
2019-12-07 20:15:32,181 rtobject INFO 1 execution context was created.
2019-12-07 20:15:32,187 rtobject INFO Initial active configuration set is default-set.
2019-12-07 20:15:36,259 SdoServiceAdmin WARNING No available SDO service in the factory: 
2019-12-07 20:15:36,260 SdoServiceAdmin ERROR type  already exists.
2019-12-07 20:15:36,262 SdoServiceAdmin WARNING No available SDO service in the factory: IDL:OpenRTM/ComponentObserver:1.0
2019-12-07 20:15:36,262 SdoServiceAdmin ERROR type 1cb70c4c-4b10-4ca8-bc38-d26674b9bec3 already exists.
2019-12-07 20:15:47,993 ec_worker ERROR State of the RTC is not ACTIVE_STATE.
2019-12-07 20:15:53,319 rtobject ERROR Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OpenRTM_aist\RTObject.py", line 1560, in on_execute
    ret = self.onExecute(ec_id)
  File "C:\workspacess\Change\Change.py", line 149, in onExecute
    FSR_dic.setdefault(("FSR" + str(i)), float(rawdata[i-1]))
ValueError: could not convert string to float: 

